### PR TITLE
Added context switch to grafana alert dashbaord while fetching alert

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/monitoring-integration#204

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @rishubhjain

File: tendrl/monitoring_integration/grafana/alert.py [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+                            
+                            alert_id),
+               auth=config["credentials"])
+    return resp
+
+def switch_context(org_name):
+    alert_org_id = grafana_org_utils.get_org_id(
+        org_name
+    )
+    grafana_org_utils.switch_context(json.loads(
```
Comment (rishubhjain) [inline tendrl/monitoring_integration/grafana/alert.py]: Switch context returns true and false which confirms whther the switch was successful or not. So I think you should check that before proceeding to other actions assuming that it has switched.

Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/grafana/alert.py]: ok

**Comment by GowthamShanmugam** [comment]
@rishubhjain changes done

**Pull request merged**